src/main/resources/languages/messages_en.yml

```
@@ -243,6 +243,7 @@ machines:
   full-inventory: '&eSorry, my inventory is too full!'
   in-use: '&cThis Block''s inventory is currently opened by a different Player.'
   ignition-chamber-no-flint: '&cIgnition Chamber is missing Flint and Steel.'
+  inventory-empty: '&cThe inventory is empty. No recipe has been placed inside the Dispenser.'
```

I think that message is not any better than the current one.
Keep in mind that this is something that early players are gonna see, so it should rather be worded more carefully., more encouraging. I would suggest:

```suggestion
  inventory-empty: '&6You have successfully constructed this Multiblock. Proceed by placing items inside the dispenser and click me again to craft the item you want!'
```